html/janus.js

```diff
@@ -968,7 +968,7 @@ function Janus(gatewayCallbacks) {
 				if(errorThrown === "")
 					callbacks.error(textStatus + ": Is the server down?");
 				else
-					callbacks.error(textStatus + ": " + errorThrown);
+					callbacks.error(textStatus + ": " + errorThrown.error.stack);
```

Comment:
Is this guaranteed to always exist? There are different implementations of `Janus.httpAPICall` in `janus.js`, and people can provide their own: while the default (using `Janus.useDefaultDependencies`) uses `fetch`, there's a version (using `Janus.useOldDependencies`) that uses jQuery's XHR stack instead.

From how I see it, in case `errorThrown.error` is `null` or `undefined`, this will cause an error while trying to report an actual problem.

Reply:
@rcmuniz1994 ping :slightly_smiling_face:

Reply:
@lminiero Hi! I see. In case errorThrown.error is null or undefined I think something like this will avoid the error:

```js
...
else if (errorThrown && errorThrown.error) {
   callbacks.error(textStatus + ": " + errorThrown.error.stack);
}
```

I'm going to test this solution and I'll try another way in case this one cause an error too.

Reply:
I checked the Janus.useDefaultDependencies and Janus.useOldDependencies. Now I totally get what you said, there are many error implementations.

Reply:
Thanks! I'll have a look tomorrow :+1: